### PR TITLE
Added contributing guide

### PR DESCRIPTION
**Pull request by aneshlya**

### CODE REVIEW

**Comment by Jasper-Bekkers** [comment]
This probably should also include build steps for each platform, since just checking the repo out and running cmake doesn't seem to work on windows (requires at least LLVM installed by the looks of it).

**Comment by aneshlya** [comment]
@Jasper-Bekkers , thanks for your comment! We have build instructions here https://github.com/ispc/ispc/wiki/ISPC-Development-Guide, and link to the wiki in Contributing.md.

**Comment by Jasper-Bekkers** [comment]
Thanks! I was looking for them today and never found them 👀

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,4 +61,15 @@ Describe the change you've made in the PR description, put an issue number if ex
...
 
-Squash git history to meaningful commits. One commit is responsible for one change.
+Squash git history to meaningful commits. One commit is responsible for one change for easier review.
+
+
+### Contribute a feature or maintain a target
+
+We are looking for individual, academic and institutional contributors who are interested in more significant contributions than a single fix as well. The possible list of topic include:
+- Fine-tuning existing targets, x86 and non-x86. Contributions to ARM targets are especially appreciated, as the core team doesn't have enough bandwidth to fully cover this.
+- Adding and maintaining a new hardware targets - Webassembly (WASM), RISC-V, ARM v9, etc.
```
Comment (aneshlya) [inline CONTRIBUTING.md:71]: Article "a" is not needed in "a new hardware targets"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -47,4 +61,15 @@ Describe the change you've made in the PR description, put an issue number if ex
...
+- Adding and/or maintaining a new supported OS/platform - PS4/PS5 (hey Sony, I'm calling for you!), Windows on ARM, FreeBSD, etc.
+- Adding a new optimizations or doing an experimental implementation for a novel research idea - for example, adding a [superoptimization](https://en.wikipedia.org/wiki/Superoptimization) pass.
```
Comment (aneshlya) [inline CONTRIBUTING.md:73]: Article "a" is not needed in "a optimizations"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -47,4 +61,15 @@ Describe the change you've made in the PR description, put an issue number if ex
...
-Squash git history to meaningful commits. One commit is responsible for one change.
+Squash git history to meaningful commits. One commit is responsible for one change for easier review.
+
+
+### Contribute a feature or maintain a target
+
+We are looking for individual, academic and institutional contributors who are interested in more significant contributions than a single fix as well. The possible list of topic include:
+- Fine-tuning existing targets, x86 and non-x86. Contributions to ARM targets are especially appreciated, as the core team doesn't have enough bandwidth to fully cover this.
+- Adding and maintaining a new hardware targets - Webassembly (WASM), RISC-V, ARM v9, etc.
+- Adding and/or maintaining a new supported OS/platform - PS4/PS5 (hey Sony, I'm calling for you!), Windows on ARM, FreeBSD, etc.
```
Comment (aneshlya) [inline CONTRIBUTING.md:72]: Seems to me that the word "supported " is not needed here: "Adding and/or maintaining a new supported OS/platform"

**Comment by aneshlya** [comment]
The extended version of the contribution guide looks great! Very motivational :) I cannot stop myself from fixing some bug right now after reading it 👍

**Pull request merged**